src/functional/ffront/builtins.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from numpy import float32, float64, int32, int64
```

Comment:
We would need to implement our own definitions for these types but I think we can keep it like that for now

Reply:
agreed.